dwave_networkx/drawing/chimera_layout.py

```diff
@@ -230,87 +230,88 @@ def draw_chimera(G, **kwargs):
 
 
 def draw_chimera_embedding(G, *args, **kwargs):
-    """Draws an embedding onto the chimera graph G, according to layout.
+    """Draw an embedding onto the chimera graph ``G``, according to the Chimera layout.
 
-    If interaction_edges is not None, then only display the couplers in that
-    list.  If embedded_graph is not None, the only display the couplers between
-    chains with intended couplings according to embedded_graph.
+    If the ``interaction_edges`` parameter is not None, then only display the couplers in that
+    list. If the ``embedded_graph`` parameter is not None, then only display the couplers between
+    chains with intended couplings according to the ``embedded_graph`` parameter.
 
     Parameters
     ----------
     G : NetworkX graph
-        Should be a Chimera graph or a subgraph of a Chimera graph.
+        :term:`Chimera` :term:`graph` or a :term:`subgraph` of a Chimera graph.
 
     emb : dict
-        A dict of chains associated with each node in G.  Should be
-        of the form {node: chain, ...}.  Chains should be iterables
-        of qubit labels (qubits are nodes in G).
+        A dict of chains associated with each node in ``G`` and
+        of the form ``{node: chain, ...}``. Chains are iterables
+        of qubit labels (qubits are nodes in ``G``).
 
     embedded_graph : NetworkX graph (optional, default None)
-        A graph which contains all keys of emb as nodes.  If specified,
-        edges of G will be considered interactions if and only if they
-        exist between two chains of emb if their keys are connected by
-        an edge in embedded_graph
+        Graph which contains all keys of the ``emb`` parameter as nodes. If specified, then
+        the edges of ``G`` will be considered interactions if and only if they
+        exist between two chains of the ``emb`` parameter and if their keys are connected by
+        an edge in the ``embedded_graph`` parameter.
```

Comment:
Perhaps it is helpful to use a "the ``x`` parameter" for the first usage to call attention that the formatted text is params listed and described here, but three uses adds text with decreasing marginal utility